state/accountsDB.go

```diff
@@ -658,6 +658,36 @@ func (adb *AccountsDB) GetExistingAccount(address []byte) (vmcommon.AccountHandl
 	return acnt, nil
 }
 
+// GetAccountFromBytes returns an account from the given bytes
+func (adb *AccountsDB) GetAccountFromBytes(address []byte, accountBytes []byte) (vmcommon.AccountHandler, error) {
```

Comment:
is this function a getter or a "creator"? I think it could be renamed to `CreateAccountFromBytes`

Reply:
It does not create a new account, but rather builds one from existing bytes. I would leave it like this. :grin: